app/views/recipes/new.html.haml

```diff
@@ -8,7 +8,7 @@
   = form.fields_for :yeasts do |yeast_fields|
     %p
       = yeast_fields.text_field :name, placeholder: 'Yeast name'
-      = yeast_fields.number_field :attenuation, placeholder: 'Yeast attenuation', min: 0.5
+      = yeast_fields.number_field :attenuation, placeholder: 'Yeast attenuation'
```

Comment:
`_form` partial might be worth introducing soon so these kind of changes only need to be made in one place vs. both of the new and edit templates